site/app/models/gradeable/AutoGradedVersion.php

```diff
@@ -183,6 +185,52 @@ private function loadSubmissionFiles() {
             }
         }
     }
+    public function getDockerErrorFileExist(): bool {
```

Comment:
```suggestion
/**
* @return bool
*/
    public function getDockerErrorFileExist(): bool {
```

Reply:
In cases where the return type doesn't need more specification, there's no need to have PHPDoc comments as well.  PHPDoc comments should really only be used for specifying array types and shapes in practice.